### ISSUE TITLE
SVG elements receiving focus on IE11

### ISSUE THREAD

**Issue by kaycee45** [nhsuk/nhsuk-frontend#737]
SVG images on IE11 are focusable by default, so this means that when a keyboard user navigates the NHS sites on IE11, all SVG images receive focus automatically. This should not be the case because it is an accessibility fail and it also could confuse users who depend on keyboard for navigation and assistive technologies.

Below is more details on the issue and my resolution

Steps to recreate:
1. Open any nhs.uk site e.g https://www.nhs.uk/better-health/ on IE11
2. Use Tab to navigate through SVG images and icons
3. Observe the hidden selection on SVGs

Environment where this issue occurred:
Operating system - Windows 10
Browser: Internet Explorer 11
NHS.UK frontend package version: 3.1.0

Resolution:
All SVG images/icons should have an attribute _**focusable="false"**_ assigned to them. 
e.g
```
<svg class="nhsuk-icon nhsuk-icon__arrow-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" focusable="false">
  <path d="M4.1 12.3l2.7 3c.2.2.5.2.7 0 .1-.1.1-.2.1-.3v-2h11c.6 0 1-.4 1-1s-.4-1-1-1h-11V9c0-.2-.1-.4-.3-.5h-.2c-.1 0-.3.1-.4.2l-2.7 3c0 .2 0 .4.1.6z"></path>
</svg>
```

Below is a live GIF capture of the issue

![invisible-links](https://user-images.githubusercontent.com/13226333/117006537-7aedf900-ace0-11eb-864a-6c15c147d5dc.gif)

**Comment by input-variable** [comment]
![image](https://user-images.githubusercontent.com/11876024/125195975-fbd0f000-e24f-11eb-852e-420f8286d922.png)
I'm also getting this on Brave Version 1.26.74 (Chromium: 91.0.4472.124)

**Comment by input-variable** [comment]
![9952460929915bae51be067b4055673e](https://user-images.githubusercontent.com/11876024/125196492-1e640880-e252-11eb-9fde-f3a99e92b0fe.gif)

I'm not getting this issue on IE11. It may have been fixed in one of the pull requests from may until now as "focusable=false" is present on the NHS website.

![image](https://user-images.githubusercontent.com/11876024/125196511-320f6f00-e252-11eb-839e-c77f42c86a0b.png)

**Comment by colinrotherham** [comment]
Closed by https://github.com/nhsuk/nhsuk-frontend/pull/1521

All of the SVGs included by NHS.UK frontend now include `focusable="false"`

Thanks @paulrobertlloyd
